src/main/org/firebirdsql/jdbc/FBBlob.java

```diff
@@ -448,7 +448,7 @@ boolean isNew() {
         return isNew;
     }
 
-    GDSHelper getGdsHelper() {
+    public GDSHelper getGdsHelper() {
```

Comment:
Two usages:

- Change the current transaction, to put the user transaction in FBConnection (see InternalFBConnection.java)
- Work with blob ids with FBBlob (see ExternalEngine.java). Maybe we can pass a FBConnection to FBBlob and let FBBlob use the GDSHelper?